script.lazytv/default.py

```diff
@@ -0,0 +1,1495 @@
+#!/usr/bin/python
```

Comment:
Usually a good practice to keep this file to a minimum since it is the only file not compiled to bytecode. Usually you'd move this file to a module and the entrypoint would just import and run:

```python
from resources.lib.plugin import plugin

plugin.run()
```